src/components/Accordion/Accordion.test.tsx

```diff
@@ -379,4 +379,119 @@ describe('Accordion component', () => {
       )
     })
   })
+  describe('custom headingLevel for AccordionItems', () => {
+    it('passes on the headingLevel', () => {
+      const customTestItems: AccordionItemProps[] = [
```

Comment:
For other potential reviewers:

AccordionItemProps needed to be renamed (from AccordianItem) so that it could be exported without conflict and used here so that the type inference on `headingLevel` would match instead of just being `string` which caused type errors